src/layer/vector/CircleMarker.js

```diff
@@ -51,8 +51,13 @@ export var CircleMarker = Path.extend({
 	// @method setRadius(radius: Number): this
 	// Sets the radius of a circle marker. Units are in pixels.
 	setRadius: function (radius) {
+		var oldRadius = this._radius;
 		this.options.radius = this._radius = radius;
-		return this.redraw();
+		this.redraw();
+
+		// @event move: Event
+		// Fired when the circle marker radius is changed via [`setRadius`](#circlemarker-setradius). Old and new radius are included in event arguments as `oldRadius`, `radius`.
+		return this.fire('move', {oldRadius: oldRadius, radius: this._radius});
```

Comment:
Objection. For circle marker radius is style property, and no move events needed.

Reply:
The "problem" is that `setRadius` changes the shape and that would be good to catch. Every draw / edit plugin has troubles to update the handlers when a change is made in the code. 
Looks like that (same for radius of circle markers):
![](https://user-images.githubusercontent.com/1726265/42126755-c6ea254a-7c5a-11e8-973d-76123aa7a5d4.png)

> For circle marker radius is style property

`setStyle({radius: XYZ})` calls `setRadius()` so it will fired too if the radius is changed over styles -> Which is good

Reply:
> `setRadius` changes the shape and that would be good to catch

May be good, may be not.
Someone else might want to catch change in color.
I have no objections, except it is not `move` in this case.

Reply:
Would you say it is `move` for a circle? What do you think about `radiuschange`?

Reply:
> Would you say it is `move` for a circle?

Yes, for circle it's radius defines it's coverage.
No need for `radiuschange`.

Reply:
I don't understand why `move` is ok for Circle but not for CircleMarker? They are "same" only the projection and fixed size is different.
According to the docs:
CircleMarker
> A circle of a fixed size with radius specified in pixels

Circle:
> A class for drawing circle overlays on a map

Can you please explain what is the difference for you

Reply:
CircleMarker is just point on map. It's radius is style feature.
Circle is shape, it's radius defines it's bounds.

Reply:
@mourner what do you think? For plugins it can be very useful to catch when the radius and latlng is changed especially if circle and circlemarkers are handled equal, like [here](https://jsfiddle.net/falkedesign/ygefu7s9/)